### ISSUE TITLE
"state_is_tuple" should be "_state_is_tuple"

### ISSUE THREAD

**Issue by manuelsh** [tam17aki/recurrent-batchnorm-tensorflow#3]
https://github.com/tam17aki/recurrent-batchnorm-tensorflow/blob/92c49100f9e622de7487e40467f9550c25c079f6/BN_LSTMCell.py#L234
